mtp_noms_ops/templates/security/senders_list.html

```diff
@@ -0,0 +1,127 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load mtp_common %}
+{% load security %}
+
+{% block page_title %}{{ view.title }} – {{ block.super }}{% endblock %}
+
+{% block inner_content %}
+  <form id="filter-senders" class="mtp-security-search js-FormAnalytics" method="get">
+
+    <div class="grid-row">
+        <div class="column-two-thirds">
+          {% include 'security/forms/top-area-object-list.html' %}
+        </div>
+
+        <div class="column-one-third print-hidden">
+          <div class="mtp-info-box">
+            <div class="mtp-info-box__heading">
+              {% trans 'About payment source data' %}
+            </div>
+            <p>
+              {% trans 'One person using many cards displays as many payment sources.' %}
+            </p>
+            <p>
+              {% trans "We can't check names match their cards, we can't match people by name to different cards and we can't check if email addresses are real." %}
```

Comment:
a little style note, we should always use curly quotation marks and apostrophes even when we're copying other text where this was forgotten. can you pls make it `can’t` not `can't`